tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -27,11 +26,12 @@ public String name() {
   private final AtomicInteger placeNumber = new AtomicInteger(0);
 
   // Evaluates place layer sort ordering of inputs into an integer for the sort-key field.
-  static int getSortKey(float min_zoom, int population_rank, long population, String name) {
+  static int getSortKey(int kind_rank, float min_zoom, int population_rank, long population, String name) {
```

Comment:
Because the LabelGrid considers all places, not just cities, we need to order countries and regions before localities. So accomplish that via an integer with range 0 to 3 (0 = country, 1 = region, 2 = locality, 3 = other)

Reply:
Epic, this works now!

BTW: I flipped this so it's minZoom then kindRank and added comments. All good now.